app/views/layouts/angular.html.erb

```diff
@@ -16,6 +16,13 @@
 </head>
 <body class="body-<%= controller.controller_name %>-<%= controller.action_name %>">
   <%= render "layouts/navigation" %>
+
+  <% if ENV['SHOW_MAINTENANCE_BANNER'] && ENV['SHOW_MAINTENANCE_BANNER'].to_s.downcase == "true" %>
+    <div id='banner_id'>
```

Comment:
Why not have this `div` element in the `app/views/layouts/_banner.html.erb`? Don't really mind, just wondering.